easydmarc.com.setup.json

```diff
@@ -6,7 +6,7 @@
   "version": 1,
```

Comment:
Hi, do you want to bump the version, or it was not yet deployed by anyone?

Reply:
not deployed yet. So it is ok with the current version.